Purchases/Networking/RCBackend.m

```diff
@@ -414,9 +425,21 @@ - (void)handleLoginWithResponse:(nullable NSDictionary *)response
 
 - (void)createAliasForAppUserID:(NSString *)appUserID
                withNewAppUserID:(NSString *)newAppUserID
-                     completion:(nullable void (^)(NSError * _Nullable error))completion {
+                     completion:(nullable PostRequestResponseHandler)completion {
     NSString *escapedAppUserID = [self escapedAppUserID:appUserID];
+    if (!escapedAppUserID || [escapedAppUserID isEqualToString:@""]) {
+        RCWarnLog(@"called createAlias with an empty appUserID!");
+        completion(RCPurchasesErrorUtils.missingAppUserIDError);
```

Comment:
we need to check that `completion != nil` or `NSNull` here, right?

Reply:
`nil` would no-op, but I'm not sure what `NSNull(RCPurchasesErrorUtils.missingAppUserIDError` would do

Reply:
i wasn't able to pass NSNull when i tried from a unit test, but i'll add a check to be safe anyway...

Reply:
swift will probably not let you, but we're allowing for it in objective-c in a few places right? since we're setting callbacks to nsnull

Reply:
actually it looks like we only call this directly from `RCIdentityManager` where we pass an explicit block, so i don't think this even needs to be nullable...